tests/context/server.rs

```diff
@@ -33,7 +33,7 @@ impl EchoServer {
                     port: public_port,
                     log_level: "INFO".into(),
                     relay_url: "https://relay.walletconnect.com".into(),
-                    database_url: "postgres://postgres@localhost:5432/postgres".into(),
+                    database_url: "postgres://postgres:postgres@localhost:5432/postgres".into(),
```

Comment:
Why not pass this as an env?

Reply:
Just hacking around for now. Think I found the issue

Reply:
Cant test this locally so that's why hacking up here :)